internal/operators/ocs/validations.go

```diff
@@ -165,24 +167,21 @@ func (o *ocsValidator) nodeResources(host *models.Host, cpuCount *int64, totalRA
 		return err
 	}
 
-	disks, err := o.hostApi.GetHostValidDisks(host)
-	if err != nil {
-		o.log.Error("failed to get valid disks due to ", err.Error())
-		return err
-	}
+	o.log.Info("Counting disks with driveType HDD or SSD")
```

Comment:
This will spam the logs every few seconds, not needed

Reply:
fixed